lua/lazyvim/plugins/extras/coding/codeium.lua

```diff
@@ -0,0 +1,23 @@
+return {
+
+  -- codeium cmp source
+  {
+    "nvim-cmp",
+    commit = "6c84bc75c64f778e9f1dcb798ed41c7fcb93b639", -- lock update (break codeium)
```

Comment:
@appelgriebsch This codeium [commit](https://github.com/jcdickinson/codeium.nvim/pull/84) seems to fix the problem with `nvim-cmp`, so you might want to remove the pinning of `nvim-cmp` to the specific commit. I tested it in my personal config and seems to work, but you might also want to test it out in case I missed something.

Reply:
Thanks for notifying me. Just tested and updated PR accordingly.